test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -289,70 +288,140 @@ public void PlattCalibratorOnnxConversionTest()
 
             var initialPipeline = mlContext.Transforms.ReplaceMissingValues("Features").
                 Append(mlContext.Transforms.NormalizeMinMax("Features"));
+            return (mlContext, dataView, estimators, initialPipeline);
+        }
+
+        [Fact]
+        public void PlattCalibratorOnnxConversionTest()
+        {
+            // Step 1: Test calibrator with binary prediction trainer
+            var (mlContext, dataView, estimators, initialPipeline) = GetEstimatorsForOnnxConversionTests();
             foreach (var estimator in estimators)
             {
                 var pipeline = initialPipeline.Append(estimator).Append(mlContext.BinaryClassification.Calibrators.Platt());
                 var onnxFileName = $"{estimator}-WithPlattCalibrator.onnx";
 
                 TestPipeline(pipeline, dataView, onnxFileName, new ColumnComparison[] { new ColumnComparison("Score", 3), new ColumnComparison("PredictedLabel"), new ColumnComparison("Probability", 3) });
             }
+
+            // Step 2: Test calibrator without any binary prediction trainer
+            IDataView dataSoloCalibrator = mlContext.Data.LoadFromEnumerable(GetCalibratorTestData());
+
+            var pipelineSoloCalibrator = mlContext.BinaryClassification.Calibrators
+                .Platt();
+            var onnxFileNameSoloCalibrator = $"{pipelineSoloCalibrator}-SoloCalibrator.onnx";
+
+            TestPipeline(pipelineSoloCalibrator, dataSoloCalibrator,onnxFileNameSoloCalibrator, new ColumnComparison[] { new ColumnComparison("Probability", 3) });
+
+            // Step 3: Test calibrator with a non-default Score column name and without any binary prediction trainer
```

Comment:
Why are we testing non-default Score column name?

Reply:
We're testing non-default Score column names to check that ONNX export for these calibrators can still occur and produce the same results even if its score column name is different from the default "Score". We had issue #4700 that exposed this possibility.